# DetectionInfo.msg
int32 class_id
int32 width
int32 height
int32 center_x
int32 center_y
float32 distance
